go/go-introduction/importing-exporting-functions/multiple-results.md

```diff
@@ -40,6 +40,22 @@ The above function gets two numbers inputted and then calculates the average of
 
 Functions can also return no results. A well-known function like that is the `main()` function.
 
+Here's a function that adds two numbers, but has no return:
+```go
+func add(x, y int) {
+	z := 0
+	z = x + y
+	fmt.Println(z)
+}
+
+// Callling the function:
+func main() {
+  add(2, 3)
+}
+
+// Output
+// 5
+```
```

Comment:
instead of showing a function with no specified return type, I think we should focus on teaching learners that if you specify a return type, then that function **has** to return that type